mne/source_space.py

```diff
@@ -2821,3 +2821,57 @@ def compute_distance_to_sensors(src, info, picks=None, trans=None,
     depths = cdist(src_pos, sensor_pos)
 
     return depths
+
+
+def get_decimated_surfaces(src):
+    """Get the decimated surfaces from a source space.
+
+    Parameters
+    ----------
+    src: instance of SourceSpaces
+        The object which has decimated surfaces
+
+    Returns
+    -------
+    subsurfaces: list of dicts which contain 'rr' and 'tris' keys
+    """
+    # NOTE: is there a better way to do this? Maybe raise a TypeError?
+    assert isinstance(src, SourceSpaces)
```

Comment:
see what we do elsewhere. Return a ValueError with a message saying
what is the type expected and what was the type provided.